docs/audit-log-filter/audit-log-filter-options.md

```diff
@@ -0,0 +1,391 @@
+* `SYSLOG` - plugin writes to the syslog
+
+
+### audit-log-filter-key-derivation-iterations-count-mean
+
+| Variable Name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-key-derivation-iterations-count-mean |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 60000  |
+| Minimum value | 1000 |
+| Maximum value | 2147483647 |
+
+Defines the mean value of iterations used by the password-based derivation routine used while calculating the encryption key and iv values. The actual iterations count is generated as a random number and deviates no more than 10% from this value.
+
+### audit_log_filter_max_size
+
+| Variable Name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-max-size |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 0 |
+| Minimum value | 0 |
+| Maximum value | 18446744073709551615 |
+| Unit | bytes |
+| Block size | 4096 |
+
+This option is only supported for JSON-format files.
+
+Defines pruning based on the combined size of the files:
+
+* 0 (zero), which is the default value, disables pruning based on size.
+* A value greater than 0 (zero) enables pruning based on size and defines the limit of the combined size. When the files exceed this limit, they can be pruned.
+
+The value is based on 4096 (block size). A value is truncated to the nearest multiple of the block size. If the value is less than 4096, the value is treated as 0 (zero).
+
+If the values for `audit_log_filter_rotate_on_size` and `audit_log_filter_max_size` are greater than 0, we recommend that `audit_log_filter_max_size` value should be at least seven times the `audit_log_filter_rotate_on_size` value. A lower value generates a warning to the error log and may remove all of the rotated logs each time.
+
+Pruning requires the following options:
+
+* `audit_log_filter_max_size`
+* `audit_log_filter_rotate_on_size`
+* `audit_log_filter_prune_seconds`
+
+### audit_log_filter_password_history_keep_days
+
+| Variable Name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-password-history-keep-days |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 0  |
+
+Defines when passwords that are archived and expired are automatically removed. The option is measured in days.
+
+Encrypted log files have passwords stored in the keyring. The plugin also stores a password history, which includes archiving and the expiration of passwords. The operation of creating a password also archives the previous password.
+
+The default value is 0 (zero). This value disables the expiration of passwords. Passwords are retained forever. 
+
+A password does not expire, despite being past the value, in case the password is used for rotated audit logs.
+
+If the plugin starts and if encryption is enabled, the plugin checks for an audit log filter encryption password. If a password is not found, the plugin generates a random password.
```

Comment:
If the plugin starts and if encryption is enabled -> If the plugin starts and encryption is enabled